CurrentMeasurementRange.c

```diff
@@ -0,0 +1,68 @@
+#include "CurrentMeasurementRange.h"
+
+int cmpfunc (const void * value1, const void * value2) 
+{
+   return ( *(int*)value1 - *(int*)value2 );
+}
+//TDD Step4:New functions added to refactor code and to reduce CCN value
+
+int isConsecutive(int diff)
```

Comment:
Consistency: This function is in camelCase while the others are alllowercase.

camelCase is easier to read :)

Reply:
Thanks for the feedback, I have reworked on it